spec/controllers/tutor_comments_controller_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'rails_helper'
+
+RSpec.describe TutorCommentsController, type: :controller do
+  describe 'endpoints' do
+    before do
+      sign_in_auth
```

Comment:
Yep I could. Not all of those tutors needed `@` scope, but I suppose it would clean it up a bit if I moved it up here.